packages/orchestration/src/index.ts

```diff
@@ -8,7 +8,9 @@ export type {
   StreamOptions,
   DocumentGroundingServiceConfig,
   DocumentGroundingServiceFilter,
-  LlmModelParams
+  LlmModelParams,
+  AzureContentSafety,
+  AzureFilterThreshold
```

Comment:
The idea is that `AzureContentSafety` defined in `orchestration-types` overrides the type generated from the specification.

Modifying the type doesn't seem to be a breaking change as in the end they would both resolve to be the same code in JS.